scripts/xPhoebus.in

```diff
@@ -4,7 +4,7 @@
 
 # To get one instance, use server mode
 ID=$(id -u)
-OPT="-server 4$ID"
+#OPT="-server 4$ID"
```

Comment:
Do you want to run "server mode" or not?  We can add the fixed the port number within "unused" range in [List of TCP and UDP port number ](https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbersl).  Maybe 4530?

Reply:
We do want to run in server mode. I added port 4530 to the xPhoebus.in file.